packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -64,13 +64,13 @@ class OverlayEntry {
     if (_opaque == value)
       return;
     assert(_overlay != null);
-    _overlay.setState(() {
+    _overlay._setEntryOpacity(() {
```

Comment:
With that name, I think I'd pass a value and have _setEntryOpacity set _opaque on the overlay entry itself.